apps/site/scripts/blog-data/generate.mjs

```diff
@@ -0,0 +1,14 @@
+import { writeFileSync } from 'node:fs';
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import generateBlogData from '../../next-data/generators/blogData.mjs';
+
+const blogData = await generateBlogData();
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+writeFileSync(
+  `${__dirname}/../../public/blog-data.json`,
+  JSON.stringify(blogData),
+  'utf8'
```

Comment:
fs supports URLs, so you can use URL resolution like

```suggestion
writeFileSync(
  new URL(`../../public/blog-data.json`, import.meta.url),
  JSON.stringify(blogData),
  'utf8'
```

Reply:
oh I see! TIL! very nice thanks! 🫶